### ISSUE TITLE
Enforce minimum size in `lotus client import`

### ISSUE THREAD

**Issue by schomatis** [filecoin-project/lotus#4130]
There is a minimum size requirement for pieces that are stored on-chain. However, the client import does not
enforce that pieces meet this requirement. A piece will be successfully imported even if it does not meet the size requirements. If a client attempts to store this piece with a miner by making a deal proposal it will be rejected due to the piece being too small.

**Comment by jennijuju** [comment]
We should first work on designing how smaller deals should be handled in the network:
- should we add paddings to the smaller deals so it match the miner deal size request?
- should we just reject those deals?

**Comment by ribasushi** [comment]
@jennijuju I believe this ticket was referring to the proofs-constrained size ( which arguably is already pretty small )
https://github.com/filecoin-project/rust-fil-proofs/issues/1231#issuecomment-663914335

**Comment by schomatis** [comment]
@dkkapur No, this came from an audit. It was just a check missing in `lotus client import`, nothing more. It was only an informational item so I filed it as a regular issue.

**Comment by dkkapur** [comment]
@schomatis - thanks! In that case, do we think it should be closed for the time being? 

@ribasushi - do you know if what you flagged above is being tracked elsewhere separately?

**Comment by dkkapur** [comment]
circling back on this - we should probably investigate this since per @schomatis's response above, this could be a legitimate UX issue for importing data for clients and should be a quick fix.

**Comment by TippyFlitsUK** [comment]
Hi 👋 

The Legacy Lotus Markets sub-system [reached EOL](https://github.com/filecoin-project/lotus/releases/tag/v1.18.0) at the end of the 31st January 2023.

This ticket is being marked as `won't fix` and closed as the Lotus Team will no longer be making any further fixes or enhancements to the legacy markets subsystem.

Please feel free to re-open this ticket in the new Boost markets sub-system repository at https://github.com/filecoin-project/boost if you feel that it is still relevant.

Many thanks 🙏
